src/main/java/com/google/javascript/clutz/Depgraph.java

```diff
@@ -118,12 +126,26 @@ private void collectFiles(boolean isRoots, List<List<?>> fileList) {
       @SuppressWarnings("unchecked")
       List<List<?>> fileProperties = (List<List<?>>) rootDescriptor.get(1);
       boolean isExterns = false;
+      boolean isGoogModule = false;
+      List<String> provides = new ArrayList<>();
       for (List<?> tuple : fileProperties) {
         String key = (String) tuple.get(0);
         if ("is_externs".equals(key) && Boolean.TRUE.equals(tuple.get(1))) {
           isExterns = true;
           break;
         }
+        if ("load_flags".equals(key)) {
```

Comment:
add some comments how does the depgraph looks likes at this point. It's an odd format will be useful to have more docs around it.

Reply:
Done.